docker-compose.yml

```diff
@@ -1168,9 +1178,11 @@ services:
 ### Docker-in-Docker ################################################
     docker-in-docker:
       image: docker:dind
+      hostname: docker-in-docker
```

Comment:
docker:dind image will automatically  generate TLS certificates with `DOCKER_TLS_SAN` env variable,  
for example:

```diff
  services:
    docker-in-docker:
-     hostname: docker-in-docker
+     environment:
+       DOCKER_TLS_SAN: DNS:docker-in-docker
```

Reference: https://github.com/docker-library/docker/blob/92d278e671f3/18.09-rc/dind/dockerd-entrypoint.sh#L8-L20

Reply:
Ok, thank you.
I've update the PR.